Utilities/src/main/java/edu/stanford/hivdb/mutations/Apobec.java

```diff
@@ -25,7 +24,9 @@
 import edu.stanford.hivdb.utilities.Cachable;
 
 public class Apobec {
-
+	
+	final private static JdbcDatabase db = JdbcDatabase.getDefault(); 
```

Comment:
The db connection shouldn't be defined (and exposed) out of the populate functions (because it is dev-only). Please revert this line change. There's no database to connect in production.

Reply:
Totally, agree. I think I accidentally left that line hanging out at the class level while moving  the static init block. Thank you for spotting it! I've the db connection back to populate function.